### DISCUSSION TITLE
New project names ideas

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/LinkFree#7320] [Ideas]
We are thinking of changing the LinkFree name, and would love to hear your ideas!

*This discussion is an exercise only and we do not guarantee the most voted names will be used*

**MUST HAVE**
- exact same name available on social platforms (no underscore or hyphens)
- `.com`, `.io` domains must be available 
- short as possible

**MUST NOT**
- contain `dev`,`link` in the name
- have existing companies or projects with the same name (please google and check)
- be similar to existing names, (for example `LinkFree` is too similar to `LinkTree`)

**Please create a new thread for each name, so people can vote**

**Comment by loftwah** [comment]
[CliqBio.com](https://oneword.domains/d/CliqBio.com) and [CliqBio.io](https://oneword.domains/d/CliqBio.io) are both available.

**Reply by eddiejaoude** [reply]
Oh those are interesting 👍 thanks Dean!

**Reply by loftwah** [reply]
[SynkBio.com](https://oneword.domains/d/SynkBio.com) and [SynkBio.io](https://oneword.domains/d/SynkBio.io) are also both available.

**Reply by eddiejaoude** [reply]
I like these also, thank you!

**Reply by eddiejaoude** [reply]
`CliqBio` would be great but the name is already taken by a similar project https://cliq.bio 😢

**Reply by ikrishg** [reply]
That is sad, cliqbio was a amazing name

**Comment by SagarSharma2809** [comment]
ToGetMe.io and ToGetMe.com ?

**Reply by eddiejaoude** [reply]
Oh these are interesting 👍 thank you

**Reply by eddiejaoude** [reply]
I wonder if `ToGet.me` is available 🤔

**Reply by SagarSharma2809** [reply]
ToGet.me is not available

**Reply by loftwah** [reply]
GoGet.me?

I tried a few similar names and they were all taken.

**Reply by eddiejaoude** [reply]
Thanks for checking! Probably for the best, not sure if it sounds like a dating app lol

**Comment by KQuiggins** [comment]
OpenDevConnect

**Reply by eddiejaoude** [reply]
Thank you, but we are keen for it not to contain `dev` - please see the guidelines in the description

**Reply by KQuiggins** [reply]
Sorry about that. This is harder than it sounds🤣
I have went through CodeFlow & CodeWave the domain names are available but there are sites out there with similar names.
Back to the drawing board 🤔

**Reply by eddiejaoude** [reply]
yes it is super difficult 👍

**Reply by loftwah** [reply]
You can use an API to check for availability if you give it a list of names. I wrote a Ruby script to do it with any names I have in a text file.

**Reply by ikrishg** [reply]
Yes it is super difficult! I'll write a js script 0.0

**Comment by iriteshmishra** [comment]
Aboutme . io?

**Reply by eddiejaoude** [reply]
But it is not available?

**Comment by CBID2** [comment]
Here’s mine @eddiejaoude! :)
1. HelloI’m.io 
2. Meetme.io
3. LinkRoot.io

**Reply by eddiejaoude** [reply]
`linkroot .io` is available but has `link` in the name and we were trying to avoid that (see the description)

I do like `root` idea though 👍

**Reply by ikrishg** [reply]
I don't know if there can be `'` in domain names, there can only be `-` and alphanumeric characters. About meetme it looks like a dating app 😅

**Comment by ibilalkayy** [comment]
Here are my ideas @eddiejaoude 

1. **Bracket** (everything is in a bracket)
2. **Torch** (when you open a torch, all your details are shown up)
3. **Grip** (Your all information is gripped)
4. **History** (Your whole achievement history)
5. **RealMe** (Your real information)
6. **LinkSpace** (Inspiration from Twitter space)

**Reply by ibilalkayy** [reply]
Although I didn't check the domain but I am sure they will be helpful

**Comment by eddiejaoude** [comment]
What about `BioDrop .io` 

- domain is available
- twitter is not available

**Reply by iriteshmishra** [reply]
BioDrop sounds good 👍
Twitter :- Biodrop_io ?

**Reply by ikrishg** [reply]
BioDropHQ can be a twitter

**Reply by HammadMaqbool** [reply]
Inspire by AirDrop.

**Reply by loftwah** [reply]
It was in here the entire time :)

**Comment by eddiejaoude** [comment]
`rootbio .io`

- domain is available
- twitter is not available

**Reply by ikrishg** [reply]
GetRootBio can be our twitter

**Comment by sandramsc** [comment]
[biolync.io](https://www.domain.com/registration/?flow=domainDFE&search=biolync.com#/domainDFE/1) and [biolync.com](https://www.domain.com/registration/?flow=domainDFE&search=biolync.com#/domainDFE/1) are both available.

**Reply by eddiejaoude** [reply]
Thanks Sandra ❤️ - a name has been selected and will be shard in a few weeks with the new logo and colours

**Comment by eddiejaoude** [comment]
BioDrop was selected and has been launched! Thank you everyone for your help and support 🎉

**Reply by loftwah** [reply]
Looks awesome!
